eng/doc/fips/UserGuide.md

```diff
@@ -684,4 +1259,58 @@ When using TLS in FIPS-only mode the TLS handshake has the following restriction
   - `tls.ECDSAWithP521AndSHA512`
 
 [EVP_EncryptUpdate]: https://www.openssl.org/docs/manmaster/man3/EVP_EncryptUpdate.html
-[EVP_DecryptUpdate]: https://www.openssl.org/docs/manmaster/man3/EVP_DecryptUpdate.html
\ No newline at end of file
+[EVP_DecryptUpdate]: https://www.openssl.org/docs/manmaster/man3/EVP_DecryptUpdate.html
+[RAND_bytes]: https://www.openssl.org/docs/man3.0/man3/RAND_bytes.html

```

Comment:
I noticed some of these are `manmaster` and some `man3.0`, is there a reason?

Reply:
I suppose Bing returns `manmaster` in some cases, but the good one is `man3.0`. Fixed!